sdcm/tester.py

```diff
@@ -2292,7 +2294,14 @@ def create_materialized_view(self, ks_name, base_table_name, mv_name, mv_partiti
             query += ' AND COMPACT STORAGE'
 
         self.log.debug('MV create statement: {}'.format(query))
-        session.execute(query)
+        if filter_view_update_error:
+            with DbEventsFilter(db_event=DatabaseLogEvent.DATABASE_ERROR,
```

Comment:
please use `EventsFilter` with `extra_time_to_expiration` instead of waiting 30sec

Reply:
@fruch , `extra_time_to_expiration` is found in `EventsSeverityChangerFilter`. do you mean using it instead of `DbEventsFilter`?

Reply:
I was talking about `EventsFilter`, `EventsSeverityChangerFilter` inherits from `EventsFilter`

Reply:
fixed